build.sbt

```diff
@@ -402,8 +414,10 @@ lazy val `proxy-core` = (project in file("proxy/core"))
         "com.thesamet.scalapb" %% "scalapb-runtime" % scalapb.compiler.Version.scalapbVersion % "protobuf",
         "io.prometheus" % "simpleclient" % PrometheusClientVersion,
         "io.prometheus" % "simpleclient_common" % PrometheusClientVersion,
-        "org.slf4j" % "slf4j-simple" % Slf4jSimpleVersion
+        "org.slf4j" % "slf4j-simple" % Slf4jSimpleVersion,
         //"ch.qos.logback"                 % "logback-classic"                   % "1.2.3", // Doesn't work well with SubstrateVM: https://github.com/vmencik/akka-graal-native/blob/master/README.md#logging
+        "com.typesafe.slick" %% "slick" % SlickVersion,
+        "com.typesafe.slick" %% "slick-hikaricp" % SlickHikariVersion
```

Comment:
I agree with the slick-dependencies only in proxy-jdbc. I am looking for the best option to have it separated.
The in-memory and the jdbc should be somehow pluggable and the in-memory should live in the core. @pvlugter perhaps you have some options here.

Reply:
The common approach for Lightbend projects is to have a setting in config, that includes a fully qualified class name that can be created dynamically. So that there's something like:

```
cloudstate.proxy.entity {
  store = in-memory

  in-memory {
    store-class = "fully.qualified.class.name.for.InMemoryStore"
    another-setting-for-in-memory = something
  }
}
```

```
# in proxy-jdbc, override setting

cloudstate.proxy.entity {
  store = jdbc

  jdbc {
    store-class = "fully.qualified.class.name.for.JdbcStore"
  }
}
```

I can take a look at adding this.